src/libs/actions/Policy.ts

```diff
@@ -2378,6 +2379,7 @@ function setWorkspaceCategoryEnabled(policyID: string, categoriesToUpdate: Recor
                             pendingFields: {
                                 enabled: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
                             },
+                            pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
                         };
```

Comment:
I don't think we need both of these. We either define `pendingFields` or `pendingAction`. I think in this case we want `pendingField`